snowpack/src/commands/dev.ts

```diff
@@ -85,6 +85,16 @@ interface FoundFile {
 
 const FILE_BUILD_RESULT_ERROR = `Build Result Error: There was a problem with a file build result.`;
 
+const COMMON_HEADERS = {
+  'Access-Control-Allow-Origin': '*',
+  'Accept-Ranges': 'bytes',
+  'Content-Type': 'application/octet-stream',
+  'Cross-Origin-Embedder-Policy': 'require-corp',
+  'Cross-Origin-Opener-Policy': 'same-origin',
+  'Cross-Origin-Resource-Policy': 'cross-origin',
+  Vary: 'Accept-Encoding',
+} as const;
```

Comment:
```suggestion
}
```
let’s not assert `as const` here (and elsewhere); that’s not a very descriptive type